docs/stack-navigator.md

```diff
@@ -275,3 +275,31 @@ const ModalNavigator = createStackNavigator(
 ```
 
 Header transitions can also be configured using `headerLeftInterpolator`, `headerTitleInterpolator` and `headerRightInterpolator` fields under `transitionConfig`.
+
+#### Specifying the transition mode for a stack's screens explicitly
+
+We can’t set the `StackNavigatorConfig`’s `mode` dynamically. Instead we are going to use a custom `transitionConfig` to render the specfific transition we want - modal or card - on a screen by screen basis.
```

Comment:
nit: you seem to be using a different apostrophe character than the rest of the docs (can’t vs can't) - can you use ' ?

Also, I think what Brent had in mind is, indeed, being more explicit about what the third param to `defaultTransitionConfig` is. ie. let's extract it into a variable with a nice name (`isModal` or `shouldUseModalTransition` or similar) and then pass it into `defaultTransitionConfig`. 

Thanks!

Reply:
Done!